app/controllers/OnlineTestController.scala

```diff
@@ -45,34 +46,34 @@ case class OnlineTestExtension(extraDays: Int)
 case class UserIdWrapper(userId: String)
 
 object OnlineTestController extends OnlineTestController {
+  override val applicationRepository: GeneralApplicationRepository = applicationRepository
   override val onlineRepository: OnlineTestRepository = onlineTestRepository
   override val onlineTestingService: OnlineTestService = OnlineTestService
   override val onlineTestExtensionService: OnlineTestExtensionService = OnlineTestExtensionService
-  override val onlineTestPDFReportRepo: OnlineTestPDFReportRepository = onlineTestPDFReportRepository
 }
 
 trait OnlineTestController extends BaseController {
 
+  val applicationRepository: GeneralApplicationRepository
   val onlineRepository: OnlineTestRepository
   val onlineTestingService: OnlineTestService
   val onlineTestExtensionService: OnlineTestExtensionService
-  val onlineTestPDFReportRepo: OnlineTestPDFReportRepository
 
   import Commands.Implicits._
 
   def getOnlineTest(userId: String) = Action.async { implicit request =>
-    onlineTestingService.getOnlineTest(userId).map { onlineTest =>
-      Ok(Json.toJson(onlineTest))
+
+    onlineTestingService.getPhase1TestProfile(userId).map {
+      case Some(phase1TestProfile) => Ok(Json.toJson(phase1TestProfile))
+      case None => NotFound
     } recover {
```

Comment:
This method needs a return type according to our new style guidance (Intellij will add it for you)